app/build.gradle

```diff
@@ -50,14 +58,17 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.8.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.6.0'
```

Comment:
Check the cirrus warning about version, is 2.6.1 still compatible?

Reply:
It is a patch difference and it still work on local and it pass the tests so I think it's good for now